### PR TITLE
ROX-15441: Add anomalous/baseline sections under flows tab

### PR DESCRIPTION
**Pull request by sachaudh**
## Description

This PR separates the table under the `Flows` tab into two sections: `Anomalous` and `Baseline`

Screenshot 2023-03-22 at 11 14 02 AM
Screenshot 2023-03-22 at 11 14 10 AM
Screenshot 2023-03-22 at 11 14 22 AM

### CODE REVIEW

**Review by zhenpesky — APPROVED** [review]

File: ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ type FlowsTableProps = {
     flows: Flow[];
     numFlows: number;
     expandedRows: string[];
-    setExpandedRows: React.Dispatch<React.SetStateAction<string[]>>;
+    setExpandedRows?: React.Dispatch<React.SetStateAction<string[]>>;
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx:32]: I'm curious why you marked this prop as optional, even though there is no other change that I can see in this PR where it's no longer passed.

File: ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx [hunk 2]
```diff
@@ -14,6 +16,8 @@ import {
     ToolbarItem,
 } from '@patternfly/react-core';
 
+import useSelectToggle from 'hooks/patternfly/useSelectToggle';
+import pluralize from 'pluralize';
```
Comment (vjwilson) [inline ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx:20]: nit: We try to keep 3rd-party imports above our own imports.

**Review by vjwilson — APPROVED** [review]
Just a couple of little things to check.

File: ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/NetworkGraph/common/FlowsTable.tsx:32]: Oh, I think I tried to remove it but then decided not to do it in this PR. We technically aren't showing expanded rows anymore so I think there's some dead code in our Network Graph code right now

**Review by zhenpesky — APPROVED** [review]

**Pull request merged**